packages/hub/src/users.ts

```diff
@@ -77,4 +70,7 @@ Client.prototype.listThreads = async function (ctx?: Context) {
   })
 }
 
-export { Client }
+/**
+ * Clients is a web-gRPC wrapper client for communicating with the web-gRPC enabled Threads API.
+ */
+export { Client, UserAuth, KeyInfo }
```

Comment:
i want to ensure that these types (and maybe others) are available to hub users. i'll likely split them into a new js-textile package so i can document them easier here too